service/common/src/main/java/org/apache/polaris/service/events/AfterTableCommitEvent.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.service.events;
+
+import org.apache.iceberg.TableMetadata;
+
+/**
+ * Emitted after Polaris performs a commit to a table. This is not emitted if there's an exception
+ * while committing.
+ *
+ * @param base The old metadata.
+ * @param metadata The new metadata.
+ */
+public record AfterTableCommitEvent(TableMetadata base, TableMetadata metadata) implements PolarisEvent {}
```

Comment:
```suggestion
public record AfterTableCommitEvent(TableMetadata base, TableMetadata new) implements PolarisEvent {}
```

Reply:
I'd suggest rename it to `AfterTableCommitted` or `AfterTableCommittedEvent` to indicate that a table commit successfully, because it is for that. We don't emit this event in case of any commit failure. I guess we don't need the word "After", it can be just `TableCommitted`.